feature-libs/product-configurator/rulebased/components/price/configurator-price.component.service.spec.ts

```diff
@@ -0,0 +1,61 @@
+import { TestBed } from '@angular/core/testing';
+import { ConfiguratorPriceService } from './configurator-price.component.service';
+import { DirectionMode, DirectionService } from '@spartacus/storefront';
+
+let direction: DirectionMode;
+class MockDirectionService {
+  getDirection(): DirectionMode {
+    return direction;
+  }
+}
+
+describe('ConfiguratorPriceService', () => {
+  let classUnderTest: ConfiguratorPriceService;
+
+  beforeEach(() => {
+    TestBed.configureTestingModule({
+      providers: [
+        {
+          provide: DirectionService,
+          useClass: MockDirectionService,
+        },
+      ],
+    });
+    direction = DirectionMode.LTR;
+    classUnderTest = TestBed.inject(ConfiguratorPriceService);
+  });
+
+  it('should create', () => {
+    expect(classUnderTest).toBeDefined();
+  });
+
+  describe('compile formatted value', () => {
+    it('should not add sign to zero value', () => {
```

Comment:
Could you explain to me the tet description?

Reply:
if the price is zero, a sign (+/-) shall not be added